src/modules/shared/components/MediaCardList/MediaCardList.tsx

```diff
@@ -125,25 +132,46 @@ const MediaCardList: FC<MediaCardListProps> = ({
 		];
 	};
 
+	const checkLocallyAvailable = (item: IdentifiableMediaCard) => {
+		const userHasAccess = activeVisits?.items.find(
+			(visit) => visit.spaceSlug === item.maintainerSlug
+		);
+		const itemHasNoVisitLicense = !item.licenses?.includes(
```

Comment:
Misschien kan je, gelijkaardig aan de `useHasAnyGroup(Group.X);` een hook maken die licenses kan afchecken. Je kan die hook dan buiten de functie aanspreken en dan verder binnen deze fn gebruiken.

```
// buiten functie
const itemHasNoCPLicense = useHasAnyLicense(IeObjectLicense.INTRA_CP_CONTENT);
```

Reply:
Hoe zie je dat precies? Want dan moet ik het item ook meegeven. Bij de andere hooks wordt de user opgeslaan in de store en kan je die idd makkelijk checken, maar hier voor 1 item uit een lijst van items lijkt me dat meer code dan nodig?